action/bureaucracy.php

```diff
@@ -155,11 +155,13 @@ public function handle_save(Doku_Event $event, $param) {
                 );
 
                 // trigger meta data rendering to set page title
-                // expire the cache in order to correctly render the struct header on the first page visit
-                p_get_metadata($id, array('cache' => 'expire'));
+                p_get_metadata($id);
```

Comment:
This line seems to not do anything anymore now. Can we remove it?

Reply:
Yes. I left it because it was present in the original file (before my first commit on this issue). But the current commit doesn't solve the problem completely. Usually it works but for some cases not (I mean that the page don't have the struct headers visible on the first revision). Unfortunately I didn't find the pattern yet.